agent/src/main/java/reactor/blockhound/BlockingOperationError.java

```diff
@@ -16,13 +16,24 @@
 
 package reactor.blockhound;
 
-public class BlockingOperationError extends Error {
+public final class BlockingOperationError extends Error {
 
     private static final long serialVersionUID = 4980196508457280342L;
 
-    public BlockingOperationError(String message) {
-        super(message);
+    private final BlockingMethod blockingMethod;
+
+    public BlockingOperationError(BlockingMethod blockingMethod) {
+        super();
+        this.blockingMethod = blockingMethod;
+    }
+
+    public BlockingMethod getBlockingMethod() {
+        return blockingMethod;
     }
 
+    @Override
+    public String toString() {
+        return super.toString() + ": " + String.format("Blocking call! %s", blockingMethod);
```

Comment:
I think should be just be `return super.toString + ": " + blockingMethod;` omitting the string `Blocking call!` from the error message.
The name of the error `BlockingOperationError` is descriptive enough.

@bsideup thoughts ?

If yes, a few test cases need to fixed which have asserts like `assertThat(e).hasMessageEndingWith("Blocking call! " + method);`

Reply:
I agree that the prefix is unnecessary. Although now I have a dilemma:

Before, the message was the only way to check if the blocking call is really reported (and thanks a lot for fixing it!)
Some integrations (like [Netty's](https://github.com/netty/netty/pull/9687/files#diff-7b9aee0f390b3827e4d1bf44e0c21d5aR52)) did use it.

If we remove it, they will need to adjust the tests when updating to BlockHound, which kinda feels like a breaking change.
On the other hand, strings should be informative and making assertions on them cannot guarantee that they will stay like that forever.

WDYT about keeping the prefix, releasing a version (so that they can update and start asserting the type) and then removing the prefix?

Reply:
Definitely what you said.

I guess another `Issue` needs to be created to keep track that the `Readme` should be updated, after a new version with the removed prefix is released.